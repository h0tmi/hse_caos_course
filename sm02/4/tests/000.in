0 0 100
0 1 200
1 1
1 2
1 2
0 5 10
1 6
1 7
1 8
