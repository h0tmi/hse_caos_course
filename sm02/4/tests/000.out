100
200
200
10
100
-1
